crates/astria-composer/src/searcher/mod.rs

```diff
@@ -77,7 +76,11 @@ impl Searcher {
     /// Errors are returned in the following scenarios:
     /// + failed to connect to the eth RPC server;
     /// + failed to construct a sequencer clinet
-    pub(super) fn from_config(cfg: &Config) -> eyre::Result<Self> {
+    pub(super) fn from_config(
+        cfg: &Config,
+        serialized_rollup_transactions_tx: Sender<SequenceAction>,
```

Comment:
nit: i would qualify these as `mpsc::Sender/Receiver` since we have `watch::Sender/Receiver`s here as well.

Reply:
sure sure